docs/api/actions.md

```diff
@@ -51,15 +51,18 @@ Simulates multiple taps on the element at its activation point. All taps are app
 await element(by.id('tappable')).multiTap(3);
 ```
 
-### `longPress(duration)`
+### `longPress(duration, point)`
```

Comment:
Don't forget to revisit if/when you change the API. 🙏

Reply:
done